itom-packages/apps/roughnessEvaluator/profile_roughness.py

```diff
@@ -63,7 +62,7 @@ def loadItem(self, dataObj: dataObject):
         ):
             ui.msgWarning(
                 "wrong unit",
-                "Value and horizontal axis unit must be nm, �m or mm",
+                "Value and horizontal axis unit must be nm, �m or mm",
```

Comment:
the special character should be µm, if the file is stored as utf8. Please check this (e.g. with notepad++). Check encoding and that the µ-sign is correct.